docs/MO_DG/prepare_model/Config_Model_Optimizer.md

```diff
@@ -46,21 +46,13 @@ install_prerequisites_caffe.sh
 ```
 install_prerequisites_caffe.bat
 ```
-*   For TensorFlow\* 1.x on Linux:
-```
-install_prerequisites_tf.sh
-```
-*   For TensorFlow 1.x on Windows:
-```
-install_prerequisites_tf.bat
-```
 *   For TensorFlow\* 2.x on Linux:
```

Comment:
please recover these lines, otherwise, it will be confusion as if TF1 models are no longer supported

Reply:
Done.